src/sentry/api/serializers/models/organization.py

```diff
@@ -205,11 +175,61 @@ def serialize(self, obj, attrs, user, access):
             'scrapeJavaScript': bool(obj.get_option('sentry:scrape_javascript', SCRAPE_JAVASCRIPT_DEFAULT)),
             'trustedRelays': obj.get_option('sentry:trusted-relays', TRUSTED_RELAYS_DEFAULT) or [],
         })
-        context['teams'] = serialize(team_list, user, TeamSerializer())
-        context['projects'] = serialize(project_list, user, ProjectSummarySerializer())
         context['access'] = access.scopes
         context['pendingAccessRequests'] = OrganizationAccessRequest.objects.filter(
             team__organization=obj,
         ).count()
         context['onboardingTasks'] = serialize(onboarding_tasks, user, OnboardingTasksSerializer())
         return context
+
+
+class DetailedOrganizationSerializerWithProjectsAndTeams(DetailedOrganizationSerializer):
```

Comment:
This is a bit wordy, but it's explicit?

Reply:
Yeah, I can't think of any way to make it more concise. But imo it's better than `LightDetailed...`